src/LoggingManager.cpp

```diff
@@ -107,6 +107,7 @@ void LoggingManager::updateConditionManagerLog(bool first_time, m_clock::time_po
     
     for (size_t id = 0; id < m_conditions.getNHVPMT(); id++) {
         hv_values[ "hv_" + std::to_string(id) + "_setValue" ] = m_conditions.getHVPMTSetValue(id);
+        hv_values[ "hv_" + std::to_string(id) + "_readValue" ] = m_conditions.getHVPMTReadValue(id);
```

Comment:
This is right when clicking "propagate" while we already started the condition logger. The readValue is more relevant when starting the logger. It can e.g. allow to see if the run was started with a readValue close to the set one. I don't think having this hurts, on the other hand it is not crucial. If you prefer we can remove it.